### PR TITLE
{lang}[foss/2015a] OCaml 4.02.3 + extensions (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
fleshed out from #1056 

requires https://github.com/hpcugent/easybuild-easyblocks/pull/467

### CODE REVIEW

File: easybuild/easyconfigs/o/OCaml/OCaml-4.02.3-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+ocaml_sourcedir = name.lower() + '-' + '.'.join(version.split('.')[0:-1])
+opam_ver = '1.2.2'
+source_urls = [
+    'http://caml.inria.fr/pub/distrib/%s' % ocaml_sourcedir,
+    'https://github.com/ocaml/opam/releases/download/%s' % opam_ver,
+]
+sources = [
+    'ocaml-%(version)s.tar.gz',
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OCaml/OCaml-4.02.3-foss-2015a.eb]: `SOURCELOWER_TAR_GZ`?

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/359ec6c9d855028865dd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/97f63c4777987325e82b for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**